### PR TITLE
Lesson 1.1

### PR DESCRIPTION
**Pull request by npnote8**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React from "react";
+
+let todoList = [];
```
Comment (royemosby) [inline src/App.js]: Although your code works, it is common convention to keep all variables and functions related to a component inside of the component, regardless if it is a functional component or class-based component. We will run across an exception to this later in the course. But for now,I recommend placing your todoList object before your return statement.

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map((item) => (
+          <li key={item.id}>{item.title}</li>
```
Comment (royemosby) [inline src/App.js:14]: Short and clear to read- great work!

File: src/index.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
...
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const container = document.getElementById("root");
+const root = createRoot(container);
+root.render(<App />);
```
Comment (royemosby) [inline src/index.js:6]: I applaud your removal of strict mode and do the same on some of my projects. Just make sure you understand the ramifications of this going forward. There is a fair chance we will talk about strict mode and how it affects some of the characteristics of useEffect.

**Review by royemosby — APPROVED** [review]
Great start overall! No need to have a re-review performed but I do ask you make the suggested update to your variable location prior to starting 1.2.

File: src/App.js (hunk 1 above)
Comment (npnote8) [reply src/App.js]: Placed the todoList object before the return statement.
Thank you!

**Pull request merged**